capabilities/weather/get-weather/profile.supr

```diff
@@ -0,0 +1,30 @@
+name = "weather/get-weather"
+version = "1.0"
+
+"""
+GetWeather usecase
+Get the current weeather
+"""
```

Comment:
This should also be user descriptive

eg
<img width="1533" alt="image" src="https://user-images.githubusercontent.com/959390/133149178-c3f6c093-5eae-4609-ae04-b35ba4d58095.png">


```suggestion
"""
Get Weather
Get Current weather for the city and in chosen units.
"""
```